pkg/node/volumemgr.go

```diff
@@ -474,33 +474,47 @@ func (m *VolumeManager) handleRemovingStatus(ctx context.Context, volume *volume
 		"volumeID": volume.Name,
 	})
 
-	var (
-		err       error
-		newStatus string
-	)
-	if err = m.getProvisionerForVolume(&volume.Spec).ReleaseVolume(volume.Spec); err != nil {
+	newStatus, err := m.performVolumeRemoving(ctx, volume)
+	if err != nil && newStatus == "" {
+		return ctrl.Result{Requeue: true}, err
```

Comment:
Is it possible to address `codecov` warnings?

Reply:
add UT for overridden health

Reply:
But it's need to fail k8sclient.Update function in the test above. Unfortunately, I can't find any simple way to do it, only creating another interface. Did you ever met references for this?